09_dynamic_programming/javascript/01_longest_common_subsequence.js

```diff
@@ -1,7 +1,79 @@
-if (word_a[i] === word_b[j]) {
-  // The letters match
-  cell[i][j] = cell[i-1][j-1] + 1;
-} else {
-  // The letters don't match
-  cell[i][j] = Math.max(cell[i-1][j], cell[i][j-1]);
+
```

Comment:
this file has some weird indentation and blank lines, could you fix the formatting? Indent to 4 spaces